readthedocs/projects/constants.py

```diff
@@ -291,10 +291,10 @@
 PROJECT_PK_REGEX = '(?:[-\w]+)'
 PROJECT_SLUG_REGEX = '(?:[-\w]+)'
 
-GITHUB_REGEXS = [
-    re.compile('github.com/(.+)/(.+)(?:\.git){1}$'),
-    re.compile('github.com/(.+)/(.+)'),
-    re.compile('github.com:(.+)/(.+)\.git$'),
+GITHUB_REGEX_BASES = [
+    '{}/(.+)/(.+)(?:\.git){{1}}$',
```

Comment:
Not sure if this was on purpose, but it was `{1}` and now it's `{{1}}`. I suppose the original was correct.

Reply:
no this change is needed, because now we'll format the string to get the regex, hence the double quotation actually keeps it as it was beforehand (aka literal ``{``)